### PR TITLE
feat: billing plans

### PR DESCRIPTION
**Pull request by RB-Isiaq**
## What does this PR do?

This PR is all about Billing plans.

## How should this be manually tested?

Send a POST request to `/api/v1/billing-plans` to seed the data in the database.

Send a GET request to `/api/v1/billing-plans` to get all plans.

Send a GET request to `/api/v1/billing-plans/:id` to get a single plan.

## Checklist of what you did

- [x] Implemented endpoint to seed billing plans to the db.
- [x] Implemented endpoint to get all billing plans.
- [x] Implemented endpoint to get a single billing plan.
- [x] Returned success message with the right response payload.
- [x] Added unit tests for the billing plan service

## Linked Issue
#456 & #457  &  #458

### CODE REVIEW

**Comment by RB-Isiaq** [comment]
@Homoakin619 @TheCodeGhinux @davidolanrewaju  Please review

File: src/modules/billing-plans/billing-plan.controller.ts
Comment (Heba-WebDev) [inline src/modules/billing-plans/billing-plan.controller.ts:1]: Why '@skipAuth()' on create a billing endpoint?

Comment (RB-Isiaq) [reply src/modules/billing-plans/billing-plan.controller.ts:1]: @Heba-WebDev the route is meant to seed the plans to the db, there's only 4 billing plans.
So reason why the route isn't taking in any payload.
You can see check the create billing-plan service for reference 

Thanks.

Comment (RB-Isiaq) [reply src/modules/billing-plans/billing-plan.controller.ts:1]: The skipAuth has been removed @Heba-WebDev

**Review by Heba-WebDev — APPROVED** [review]

**Review by davidolanrewaju — APPROVED** [review]

**Review by rugue — APPROVED** [review]

**Pull request merged**